core/src/state/world_state.rs

```diff
@@ -100,16 +101,24 @@ pub struct WorldState {
     logical_time: u64,
     pub clusters: BTreeMap<ClusterName, ClusterState>,
     listeners: BTreeMap<u64, Sender<()>>,
+    last_heartbeat: Option<OffsetDateTime>,
```

Comment:
hmmm, why do we use both time and chrono?

we should pick one, ok with this for now though.

Reply:
I agree, but in this case we get an `OffsetDateTime` and don't have a need to convert it yet, so let's defer for now. `time` is a dependency via `async_nats` either way, and the value we get back is an `OffsetDateTime` either way.